hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlBasicTest.java

```diff
@@ -714,7 +717,11 @@ protected AbstractPojo(long val) {
             timeVal = timestampVal.toLocalTime();
 
             tsTzDateVal = new Date();
-            tsTzCalendarVal = (GregorianCalendar) GregorianCalendar.getInstance();
+            // GregorianCalendar with different timezones has different serialized length.
+            // It seems to vary between approximately 1000 and 2000 bytes.
+            // Use specific timezone to decrease variations between environments.
+            // Some tests (e.g. HD) have limited amount of memory which can be exceeded by longer values.
+            tsTzCalendarVal = (GregorianCalendar) GregorianCalendar.getInstance(UTC);
```

Comment:
as an alternative fix, maybe, we can just increase HD memory for the test.

Reply:
that should also help, but I prefer tests that do not depend on their environment

Reply:
Confirmed - I am in UTC and when I run without this fix, it still breaks. So adding more HD memory - seems like it will just lead to another break in the future.